### PR TITLE
[add] Homepage component

### PR DESCRIPTION
**Pull request by edbella**
This is the homepage that the user sees on landing on the url.

### CODE REVIEW

File: src/components/homepage/components/Homepage.jsx [hunk 1]
```diff
@@ -0,0 +1,86 @@
+import React from "react";
+import { Helmet } from "react-helmet";
+import { Link } from "react-router-dom";
+import { Row, Col } from "antd";
+import { Title, Helmet_Strings, Main_Content } from "../constants";
```
Comment (enyeInc) [inline src/components/homepage/components/Homepage.jsx]: All the constants should be all caps. so...
```
import { TITLE, HELMET_STRINGS, MAIN_CONTENT } from '../constants';
```

File: src/components/homepage/components/Homepage.jsx [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+      <meta name={META_VIEWPORT} content={META_VIEWPORT_CONTENT}></meta>
+      <title itemProp="name" lang="en">
+        {PAGE_TITLE}
+      </title>
+      <link rel="canonical" href={CANON_LINK} />
+    </Helmet>
+  );
+};
+
+const Page_Content = () => {
```
Comment (enyeInc) [inline src/components/homepage/components/Homepage.jsx]: that is not how you name a component. it should be `PageContent`

File: src/components/homepage/components/Homepage.jsx [hunk 3]
```diff
@@ -0,0 +1,86 @@
+import React from "react";
+import { Helmet } from "react-helmet";
+import { Link } from "react-router-dom";
+import { Row, Col } from "antd";
+import { Title, Helmet_Strings, Main_Content } from "../constants";
+
+const Page_Helmet = () => {
```
Comment (enyeInc) [inline src/components/homepage/components/Homepage.jsx]: rename to `PageHelmet`

File: src/components/homepage/components/Homepage.jsx [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+    TITLE_TEMPLATE,
+    TITLE_TEMPLATE_DEFAULT_TITLE,
+    CHARSET,
+    META_DESC,
+    META_DESC_CONTENT,
+    META_VIEWPORT,
+    META_VIEWPORT_CONTENT,
+    PAGE_TITLE,
+    CANON_LINK
+  } = Helmet_Strings;
```
Comment (enyeInc) [inline src/components/homepage/components/Homepage.jsx]: destructure these variables outside this function component

File: src/components/homepage/components/Homepage.jsx [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+    </main>
+  );
+};
+
+const Homepage = () => {
+  return (
+    <>
+      <Page_Helmet />
+      <Page_Content />
+    </>
```
Comment (enyeInc) [inline src/components/homepage/components/Homepage.jsx]: You defined both `PageHelmet` and `PageContent` in the same file. Do not do this. One component per file.

File: src/components/homepage/constants.js [hunk 6]
```diff
@@ -0,0 +1,61 @@
+/**
+ * Imports the Typography Components from the Ant Design  Library
+ * 
+ * @import 
+ */
+import {
+	Typography
+} from 'antd';
```
Comment (enyeInc) [inline src/components/homepage/constants.js]: do not import react component into the constants file.

File: src/components/homepage/constants.js [hunk 7]
```diff
@@ -0,0 +1,61 @@
...
+import Artwork from "../homepage/images/hero-illustration.svg";
+
+/**
+ * Decouples sub-components from Typography
+ * 
+ * Title - Exports the Title property from the Ant Design Library.
+ *
+ * @constant
+ */
+export const {
```
Comment (enyeInc) [inline src/components/homepage/constants.js]: Do not do this, use import it directly in the file that you need it in.

File: src/components/homepage/constants.js [hunk 8]
```diff
@@ -0,0 +1,61 @@
...
+	Title
+} = Typography;
+
+/**
+ * Exports React Helmet related  Strings
+ * 
+ * @constant
+ */
+
+export const Helmet_Strings = {
```
Comment (enyeInc) [inline src/components/homepage/constants.js]: HELMET_STRINGS

File: src/components/homepage/constants.js [hunk 9]
```diff
@@ -0,0 +1,61 @@
...
+	CANON_LINK: "http://mysite.com/example"
+}
+
+/**
+ * Exports Main Content related  Strings
+ * 
+ * @constant
+ */
+
+export const Main_Content = {
```
Comment (enyeInc) [inline src/components/homepage/constants.js]: MAIN_CONTENT

**Comment by enyeInc** [comment]
@edbella minor fixes, address them and code is ready to be merged

File: src/components/homepage/constants.js (hunk 9 above)
Comment (edbella) [reply src/components/homepage/constants.js]: Ok

File: src/components/homepage/constants.js (hunk 8 above)
Comment (edbella) [reply src/components/homepage/constants.js]: Ok

File: src/components/homepage/constants.js (hunk 7 above)
Comment (edbella) [reply src/components/homepage/constants.js]: Ok

File: src/components/homepage/constants.js (hunk 6 above)
Comment (edbella) [reply src/components/homepage/constants.js]: Ok

**Pull request merged**